cmd/proxy-server/main.go

```diff
@@ -39,12 +45,12 @@ var flags []cli.Flag = []cli.Flag{
 	&cli.StringFlag{
 		Name:    "tls-certificate",
 		EnvVars: []string{"TLS_CERTIFICATE"},
-		Usage:   "Certificate to present (PEM). Only valid for --server-attestation-type=none and with --tls-private-key.",
+		Usage:   "Path to TLS certificate (PEM). Only valid for --server-attestation-type=none and with --tls-private-key.",
```

Comment:
nit: since the usage is now explicitly mentioning the path, should the env var name also be changed accordingly?
Like TLS_CERTIFICATE_FILE or TLS_CERTIFICATE_PATH

Reply:
yeah i guess that's right, but it'll need an update of our deployment too. better bite the bullet now though. updating